aws/resource_aws_ram_resource_share_accepter.go

```diff
@@ -217,13 +237,13 @@ func resourceAwsRamResourceShareAccepterDelete(d *schema.ResourceData, meta inte
 	return nil
 }
 
-func resourceAwsRamResourceShareGetInvitation(conn *ram.RAM, resourceShareARN, status string) (*ram.ResourceShareInvitation, error) {
```

Comment:
Nit: For consistency across the provider codebase, we prefer these functions to take the service-specific clients.